doc/admin/config.rst

```diff
@@ -297,6 +297,12 @@ to speed up various operations::
     [redis]
     location=redis://127.0.0.1:6379/1
     sessions=false
+    sentinels=[
+        ["sentinel_host_1", 26379],
+        ["sentinel_host_2", 26379],
+        ["sentinel_host_3", 26379]
+    ]
```

Comment:
Does this actually work like this? https://docs.python.org/3/library/configparser.html states:

>  Values can also span multiple lines, as long as they are indented deeper than the first line of the value

Wouldn't this fail for the closing ] in the example?

Reply:
You are absolutely right, missed that. I will fix the docs.